public/apps/configuration/panels/audit-logging/audit-logging.tsx

```diff
@@ -64,8 +64,8 @@ function renderStatusPanel(onSwitchChange: () => void, auditLoggingEnabled: bool
               <EuiCode>elasticsearch.yml</EuiCode>. The default storage location is{' '}
               <EuiCode>internal_elasticsearch</EuiCode>, which stores the logs in an index on this
               cluster.{' '}
-              <EuiLink external={true} href="/">
-                Learn more
+              <EuiLink external={true} href={DocLinks.AuditLogsStorageDoc} target="_blank">
```

Comment:
Non-blocking, I would suggest have an `ExternalLink` component that accepts just an href prop.

Reply:
I will add a TODO